core/src/main/java/com/scalar/db/storage/cosmos/CosmosNamespace.java

```diff
@@ -0,0 +1,46 @@
+package com.scalar.db.storage.cosmos;
+
+import com.google.common.base.MoreObjects;
+import java.util.Objects;
+
+/** An entity class to store namespace metadata in a CosmosDB container */
+public class CosmosNamespace {
+  private String id;
+
+  public CosmosNamespace() {}
+
+  public CosmosNamespace(String id) {
+    this.id = id;
+  }
+
+  public void setId(String id) {
```

Comment:
This method doesn't seem to be used. How about putting `final` keyword on `id`?

Reply:
Since this object by the Cosmos SDK to unmarshal the query result of the namespaces table using Jackson. I thought this setter method was used to initialize the `id` attribute but eventually, making this object immutable works fine. Having the default constructor is needed though.
Thank you. cf. https://github.com/scalar-labs/scalardb/pull/989/commits/fc2def1faecf44e9de2ea88c07560db7693cb92a#diff-8538e3bb29b97980fc88f869c47afc4c1215b0608f81fe0143652964cbeef9fa

Reply:
Oh I didn't know that we don't need setter methods for this kind of classes. So we also don't need setter methods in `CosmosTableMetadata` and `Record`?

Reply:
@brfrn169 Good idea, let me try.

Reply:
@brfrn169 Removing the setters also works for `CosmosTableMetadata` and `Record`. I will push the changes in a separate PR.